docs/integration-guides/next-auth.mdx

```diff
@@ -69,53 +74,55 @@ You'll overwrite the _wellKnown_, _clientId_, and _clientSecret_ values later in
 
 2. Add the following content.
 
-    <Tabs groupId="nextjs" queryString>
-    <TabItem value="nextauth" label="NextAuth.js">
-    
-    ```javascript
-    import NextAuth from 'next-auth';
-    
-    export default NextAuth({
-      providers: [
-        {
-          id: 'beyondidentity',
-          name: 'Beyond Identity',
-          type: 'oauth',
-          wellKnown: process.env.BEYOND_IDENTITY_DISCOVERY,
-          authorization: { params: { scope: 'openid' } },
-          clientId: process.env.BEYOND_IDENTITY_CLIENT_ID,
-          clientSecret: process.env.BEYOND_IDENTITY_CLIENT_SECRET,
-          idToken: true,
-          checks: ['state', 'pkce'],
-          profile(profile) {
-            return {
-              id: profile.sub,
-              name: profile.sub,
-              email: profile.sub,
-            };
-          },
+  <Tabs groupId="nextjs" queryString>
```

Comment:
nit: I think @patriciamcphee would like us to indent code blocks when it's under a number like this.

Reply:
I think this is still fine. It is still indented; I just shifted it over one indentation.